packages/core/src/components/widgets/FileWidget.tsx

```diff
@@ -60,20 +81,31 @@ function extractFileInfo(dataURLs) {
     });
 }
 
-class FileWidget extends Component {
-  constructor(props) {
+type FileWidgetStateType = {
+  values: any[];
+  filesInfo: FileInfoType[];
+};
+
+class FileWidget<T, F> extends Component<
```

Comment:
Can you add JSDoc here as well... take a look at how the other, already converted widgets are documented as a guide. Bonus points if you document the other functions and types in this file.

Reply:
Missed the docs for the class and the functions